app/controllers/to_do_lists_controller.rb

```diff
@@ -2,6 +2,7 @@ class ToDoListsController < ApplicationController
 
   def index
     @lists = List.including_items
+    @listsPresentation = ListsPresentation.new @lists
```

Comment:
@vlasiak, I'm sorry for not seeing that before. Let's rename the variable in order to use snake_case convention: `lists_presentation`.